lib/actions/addLinkToProjectAction.js

```diff
@@ -1,32 +1,26 @@
-const assert = require('assert')
-const AddLinkToProjectResponse = require('./AddLinkToProjectResponse')
+/* eslint-disable class-methods-use-this */
```

Comment:
This comment is disabling the warning for the entire file. Let's just disable it for the one function it is warning about now. If we add a new method to this class in the future, we want ESLint to warn us about the same thing for that new method.

This comment would go right before the `parseParams` method definition.
```js
/* eslint-disable-next-line class-methods-use-this */
```